python_file.py

```
@@ -0,0 +1,213 @@
+
+class AgentInformation(APIView):
+    authentication_classes = (ProxyUserJWTAuthentication,)
+    permission_classes = (ProxyUserViewPermission,)
+
+    def get(self, request: Request) -> Response:
+        params = InformationParamSerializer(data=dict(request.query_params))
+        params.is_valid(raise_exception=True)
+        user = request.selected_user
+        agents = Agent.objects.filter(user=user, **params.data)
+        if not agents:
+            raise ValidationError('Agent not found')
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)